### DISCUSSION TITLE
Endless loop

### DISCUSSION THREAD

**Discussion by curkan** [austintoddj/canvas#1242] [Troubleshooting]
When you visit the pages /canvas/posts /canvas/users, the same data is loaded endlessly. Locally, there is no such behavior, errors do not fall into the console, most likely the problem is in vue, any suggestions?

https://user-images.githubusercontent.com/1681501/220284494-70fb7929-45bc-448c-a916-952203418bb2.mov

https://user-images.githubusercontent.com/1681501/220284578-19a4db30-6efb-4a81-b6e7-e45919fd2b10.mov

**Comment by austintoddj** [comment]
Can you share a link to the code where this is happening?

**Comment by austintoddj** [comment]
Closing as obsolete with the v7 rewrite.

The old Vue admin SPA is gone — v7 is a new React admin with a different data-loading model. If you still see an infinite request loop on a fresh [v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0) install, please open a new issue with network traces.
